bin/public/horizontal-split.html

```diff
@@ -1,24 +0,0 @@
-<!DOCTYPE html>
```

Comment:
This should *not* be deleted. It's the horizontal split introduced in latest version.